install-scripts/install_ipfix-rita.sh

```diff
@@ -0,0 +1,503 @@
+#!/usr/bin/env bash
```

Comment:
Looks like theres a lot of new stuff in the install script here. This will take me a bit to get through.